.gitignore

```diff
@@ -15,6 +15,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.vs/*
```

Comment:
@IsaacAbrahamson Hmm... this didn't seem to work. I think for 2 reasons:
1. The files were already committed to your remote repo
2. Even if this was done brand new, I realized that a double `*` will be needed, because the binary filenames have slashes in them, and a singe `*` does not match `/` in a pattern. 

I think the solution will be this:
When I get home, I want to bring my staging branch back to even with master, and maybe we can re-open this PR against staging so that we can merge it in and play around with it without worrying about never being able to go back. Once I do this, we can have you re-do the PR, but you'll likely have to make a new branch, so that these files won't get committed in the first place to your remote branch. 

To do that, change this line in the `.gitignore` file to the following, and commit the changes:
```
.vs/**
```

Then, create a new branch with all existing commits from this branch in the same way you normally would. How about something like:
```
git checkout -b feat/menu-css
```
Then follow the same steps to open a PR but choose my staging branch as the base branch. But hold off for now, I'll let you know when staging is fully up to date.